diff/diff_test.go

```diff
@@ -103,27 +115,27 @@ func TestParseFileDiffHeaders(t *testing.T) {
 			filename: "sample_file.diff",
 			wantDiff: &FileDiff{
 				OrigName: "oldname",
-				OrigTime: &pbtypes.Timestamp{Seconds: 1255273940},
+				OrigTime: &timestampsT[0],
```

Comment:
out of interest why not directly use time.Unix here?

Reply:
For `&time.Unix(1255273940, 0)` receiving an error: `cannot take the address of time.Unix(1255273940, 0)`.